scalameta/semanticdb-scalac/src/test/scala/scala/meta/tests/MemberSuite.scala

```diff
@@ -0,0 +1,161 @@
+package scala.meta.tests
+
+import scala.meta.internal.semanticdb.MemberMode
+import scala.meta.internal.semanticdb.SemanticdbMode
+
+class MemberSuite extends DatabaseSuite(SemanticdbMode.Slim, MemberMode.All) {
+  members(
+    """
+      |object a {
+      |  trait A {
+      |    def a = 2
+      |    def a(k: Int) = 2
+      |    val b = 3
+      |    var c = 3
+      |    lazy val d = 3
+      |    type E
+      |    class F
+      |    trait G
+      |  }
+      |  class AA {
+      |    def aa = 3
+      |  }
+      |  trait B extends AA with A
+      |  class C extends AA with A
+      |}
+    """.stripMargin,
+    """
+      |_empty_.a.A#{
+      |  G#
+      |  F#
+      |  d.
+      |  `c_=`.
+      |  c.
+      |  b.
+      |  a.
+      |  a.
+      |  E#
+      |}
+      |_empty_.a.AA#{
+      |  aa.
+      |}
+    """.stripMargin
+  )
+  symbols(
+    """
+      |import scala.util._
+    """.stripMargin,
+    """
+      |_root_.scala. => package scala
+      |_root_.scala.util. => package util.{+72}
```

Comment:
I changed the syntax to `{+72 members}` based on feedback that `+72` was not self-explanatory.